vpr/src/device/rr_graph_builder.h

```diff
@@ -44,6 +44,15 @@ class RRGraphBuilder {
     inline void set_node_type(RRNodeId id, t_rr_type type) {
         node_storage_.set_node_type(id, type);
     }
+    RRSegmentId add_rr_segment(const t_segment_inf& segment_info);
+    RRSwitchId add_rr_switch_inf(const t_rr_switch_inf& switch_info);
+    void reserve_segments(const size_t& num_segments);
+    size_t rr_segment_sizes() const;
```

Comment:
@tangxifan 
Can you Please have a look on s APIs here. Are these correct?
Because when I replaced in respective files my unit tests and R strong tests are failing.
Thanks

Reply:
They are o.k. except the name of API ``rr_segment_sizes()``
Can you change it to 
```
size_t num_rr_segments() const;
```

Thanks.

Reply:
Sure I will change them. But I am not getting why unit tests are and strong test is failing?